### PR TITLE
Words attached to links

### PR DESCRIPTION
**Pull request by shikhar360**
Some words were not attached to their respective links .

### CODE REVIEW

**Comment by shikhar360** [comment]
@haardikk21 😅 This as well

File: Decentralized-Exchange.md [hunk 1]
```diff
@@ -624,7 +624,7 @@ npm run dev
 
 Now go to `http://localhost:3000`, your app should be running 🤘
 
-Now lets install Web3Modal library(https://github.com/Web3Modal/web3modal). Web3Modal is an easy-to-use library to help developers add support for multiple providers in their apps with a simple customizable configuration. By default Web3Modal Library supports injected providers like (Metamask, Dapper, Gnosis Safe, Frame, Web3 Browsers, etc), You can also easily configure the library to support Portis, Fortmatic, Squarelink, Torus, Authereum, D'CENT Wallet and Arkane.
```
Comment (haardikk21) [inline Decentralized-Exchange.md:627]: Arkane is right here, not Arcana. Arcana is not a wallet.

File: Initial-Coin-Offering.md [hunk 2]
```diff
@@ -239,6 +239,7 @@ async function main() {
     cryptoDevsNFTContract
   );
 
+  await deployedCryptoDevsTokenContract.deployed();
```
Comment (haardikk21) [inline Initial-Coin-Offering.md:242]: Nice

**Review by haardikk21 — CHANGES_REQUESTED** [review]

**Comment by shikhar360** [comment]
@haardikk21 do review ser 😊

**Comment by yanukadeneth99** [comment]
Looks good. Nice work @shikhar360

**Pull request merged**